src/_includes/layouts/homepage.html

```diff
@@ -20,9 +20,40 @@
 	<body data-typeface="system-ui">
 		{% include partials/header.html %}
 		<main data-layout="centered-single-column">
+			<h2 class="events-title">Next event</h2>
+			<section id="next-event-section" class="events-section">
```

Comment:
`section`s should have an accessible label to be most usable for assistive tech users.

I recommend:

```suggestion
			<h2 id="next-event" class="events-title">Next event</h2>
			<section id="next-event-section" aria-labelledby="next-event" class="events-section">
```

Reply:
aaah! thank you 🙏
committed as suggested